uportal-war/src/main/java/org/jasig/portal/jgroups/protocols/DAO_PING.java

```diff
@@ -134,6 +142,7 @@ protected void getAndSavePhysicalAddress(String clusterName) {
     }
 
     protected void handleView(View view) {
+        log.warn("Change in JGroup membership: " + view.toString());
```

Comment:
Yes, they should be rare and only occur when an instance is joining or leaving. Thought WARN would be appropriate, especially if unexpected